src/analysis/special_functions/japanese_bracket.lean

```diff
@@ -0,0 +1,214 @@
+/-
+Copyright (c) 2022 Moritz Doll. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Moritz Doll
+-/
+
+import analysis.special_functions.integrals
+import analysis.special_functions.pow
+import measure_theory.integral.integral_eq_improper
+import measure_theory.integral.layercake
+import tactic.positivity
+
+/-!
+# Japanese Bracket
+
+In this file, we show that Japanese bracket $(1 + \|x\|^2)^{1/2}$ can be estimated from above
+and below by $1 + \|x\|$.
+The functions $(1 + \|x\|^2)^{-r/2}$ and $(1 + |x|)^{-r}$ are integrable provided that `r` is larger
+than the dimension.
+
+## Main statements
+
+* `integrable_one_add_norm`: the function $(1 + |x|)^{-r}$ is integrable
+* `integrable_jap` the Japanese bracket is integrable
+
+-/
+
+
+noncomputable theory
+
+open_locale big_operators nnreal filter topological_space ennreal
+
+open asymptotics filter set real measure_theory finite_dimensional
+
+variables {E : Type*} [normed_add_comm_group E]
+
+lemma jap_le_one_add_norm (x : E) : real.sqrt (1 + ∥x∥^2) ≤ 1 + ∥x∥ :=
```

Comment:
Since it is used multiple times here, and since it is also used in [https://github.com/leanprover-community/mathlib/blob/a15401cf044b6aee30b461f1e17348c1bfe902d7/src/analysis/normed_space/basic.lean#L202](here) I suggest we add:
```lean
lemma zero_lt_one_add_norm_sq (x : E) : 0 < 1 + ∥x∥^2 := by positivity
```
(in the relevant file) and then use it in `homeomorph_unit_ball` as well as in this file.